src/shared/components/VideoPreviewBase/VideoPreviewBase.tsx

```diff
@@ -158,7 +158,7 @@ const VideoPreviewBase: React.FC<VideoPreviewBaseProps> = ({
     onChannelClick(e)
   }
   const createAnchorClickHandler = (href?: string) => (e: React.MouseEvent<HTMLAnchorElement, MouseEvent>) => {
-    if (!href) {
+    if (!href || publisherMode) {
```

Comment:
```suggestion
    if (!href || isAnyVideoSelected) {
```

The anchor should still work when in publisher mode, tho we want to prevent it from navigating when any videos are being selected since it enters a "selecting videos" mode, and clicking a video cover means we want to select/deselect it